### PR TITLE
[EBPF-294] Add Oracle images to KMT

### PR DESCRIPTION
**Pull request by gjulianm**
### What does this PR do?

Adds Oracle 7.9 (x64 only), 8.9 and 9.3 to KMT.

### Motivation

Bringing the KMT test images as close to Kitchen as possible for security-agent tests.

Related: https://github.com/DataDog/ami-builder/pull/159
https://datadoghq.atlassian.net/browse/EBPF-294

### CODE REVIEW

**Comment by gjulianm** [comment]
/trigger-ci --variable RUN_ALL_BUILDS=true --variable RUN_KITCHEN_TESTS=true --variable RUN_E2E_TESTS=on --variable RUN_UNIT_TESTS=on

**Comment by gjulianm** [comment]
/trigger-ci --variable RUN_ALL_BUILDS=true --variable RUN_KITCHEN_TESTS=true --variable RUN_E2E_TESTS=on --variable RUN_UNIT_TESTS=on

File: test/new-e2e/system-probe/config/platforms.json
Comment (usamasaqib) [inline test/new-e2e/system-probe/config/platforms.json:1]: How about adding a separate file for security agent, so we do not download and set up images that are not tested?

Comment (gjulianm) [reply test/new-e2e/system-probe/config/platforms.json:1]: Yes, I think that's needed specially now with 6 extra images for the security agent. I'll do that in a separate PR.

**Review by paulcacheux — APPROVED** [review]

**Review by usamasaqib — APPROVED** [review]

**Pull request merged**